src/main/java/uk/gov/register/presentation/config/PresentationConfiguration.java

```diff
@@ -10,6 +12,10 @@
 public class PresentationConfiguration extends Configuration {
     @Valid
     @NotNull
+    private JerseyClientConfiguration jerseyClient = new JerseyClientConfiguration();
```

Comment:
Do you want to provide ClientConfiguration parameters optionally from configuration file? if yes @JsonProperty is missing and @NotNull annotation is not required. If not then any specific reason this object created in PresentationConfiguration?